content/events/2021-online-unconference/_index.md

```diff
@@ -9,77 +9,77 @@ description = "Nordic RSE – Research Software Engineers in the nordics – Unc
+No idea yet? - No problem, we will also accept last minute contributions during the event. 
 
-Please also have a look at our [time length suggestions](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md).
+Please also have a look at our [time slot suggestions](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md).
 
-Here are some examples of possible contributions:
+Here are some examples to spark ideas:
 - *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
 - *Demonstrations:* show us some tools or software that you like
 - *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
 - *Talks*: teach us something new
 - *On-going projects*: share with others something unfinished and almost working that you would like to get feedback on
 - *Something else*: is there something you would like to share with others that does not fit any of the previous points? We are looking forward to hearing about it!
 
+Ready to submit your idea? - Please use the [unconference contribution issue template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md) to let us know. It does not have to be polished yet, but so that the community gets the idea. We can polish it together if you want to :)
 
-## How to contribute ideas
+You may also come and chat with us in the [Zulip chat](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse), where we can figure out contributions together.
 
-- Would like to participate but don't really know how? Chat with us in the [Zulip chat](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse), we will figure out something together.
-- Once you have an idea for your contribution, open a github issue [here](https://github.com/nordic-rse/nordic-rse.github.io/issues/new/choose), use the *Unconference contribution* template.
-- You can also email us at <nordic-rse-organizers@neic.no>. In this case, please try to follow [this template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md)
-- Note that **you don't need to submit a long and fully polished proposal**, you can update your idea until the unconference and you will receive all the feedback and support that you need.
+Uncomfortable with github issues? - you can also email us at <nordic-rse-organizers@neic.no>. In this case, please try to follow [this template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md) as much as possible.
```

Comment:
Really? someone writing code and not confortable with issues? I would remove it.

Reply:
does really everyone have a github account? Some use other services... And some may not want to share preliminary ideas that public...

Reply:
Can leave the question out and start with "- you can also..."

Reply:
I could imagine some people who are not used to GitHub. At the same time, I can agree that we want to make the instructions as minimal and simple as possible. Do we lose too much by offering an alternative?

Reply:
In my understanding it wasn't meant so much as comfortable technically but perhaps comfortable to go public with an idea immediately. It was meant to offer alternative if somebody wants to grow an idea without immediately creating an issue on public record.